### PR TITLE
hortense: add debug_rome interactive and debug partition details

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: source/gent/tier1_hortense.rst [hunk 1]
```diff
@@ -52,6 +52,13 @@ Hortense consists of the following partitions:
        - 4x NVIDIA A100-SXM4 (80 GB GPU memory), NVLink3
        - 512 GiB RAM (~10GB/CPU core), no swap
        - 480 GB SSD local disk
+- ``dodrio/debug_rome``: interactive and debug partition:
+   - 3 virtual workernodes, each with:
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:56]: What's "virtual" about these workernodes? Are they VMs?

I would drop the `virtual`, it'll only lead to confusion.

**Review by hajgato — APPROVED** [review]

File: source/gent/tier1_hortense.rst [hunk 2]
```diff
@@ -350,6 +357,33 @@ Trying to make any changes to files that are accessed via ``/readonly`` will res
...
+
+Interactive and debug partition
+*******************************
+
+A (small) interactive and debug partition `debug_rome` is available where you can get
+quick access but only to a limited number of resources.
+The CPUs are oversubscribed by factor 4, which may lead to slower then expected run times when
+the usage is high.
+
+The nodes have one GPU that can be requested for exclusive access
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:370]: `one NVIDIA V100 GPU`

File: source/gent/tier1_hortense.rst [hunk 3]
```diff
@@ -350,6 +357,33 @@ Trying to make any changes to files that are accessed via ``/readonly`` will res
...
+(as with the GPU partitions) and also one shared GPU.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:371]:
```suggestion
(as with the GPU partitions), and also one less powerful GPU (NVIDIA Quadro P1000) that is always available but shared across all jobs on that node.

File: source/gent/tier1_hortense.rst [hunk 4]
```diff
@@ -350,6 +357,33 @@ Trying to make any changes to files that are accessed via ``/readonly`` will res
...
+To make use of the partition you can select the ``dodrio debug_rome`` option in the `Cluster` field in the
+`Interactive Apps` forms on the webportal, or from the CLI
+
+.. code:: shell
+
+    module swap cluster/dodrio/debug_rome
+    qsub job_script.sh
+
+
+There is no billing for using this partition.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:382]:
```suggestion
No credits are consumed when using this partition.

File: source/gent/tier1_hortense.rst [hunk 5]
```diff
@@ -350,6 +357,33 @@ Trying to make any changes to files that are accessed via ``/readonly`` will res
    visible via ``/readonly`` during the lifetime of the job, so you should not assume that this will be the case.
 
 
+.. _hortense_interactive_debug:
+
+Interactive and debug partition
+*******************************
+
+A (small) interactive and debug partition `debug_rome` is available where you can get
+quick access but only to a limited number of resources.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:366]: I would clarify what "limited number of resources" means (max. 8 cores in use, max N jobs running + X jobs in queue, etc.).

That duplicates some info from https://docs.hpc.ugent.be/interactive_debug, but it's important to highlight that imho.

File: source/gent/tier1_hortense.rst [hunk 6]
```diff
@@ -350,6 +357,33 @@ Trying to make any changes to files that are accessed via ``/readonly`` will res
...
+
+.. code:: shell
+
+    module swap cluster/dodrio/debug_rome
+    qsub job_script.sh
+
+
+There is no billing for using this partition.
+
+For some additional information, you can visit https://docs.hpc.ugent.be/interactive_debug/.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:384]:
```suggestion
For some additional information, see the documentation on the HPC-UGent Tier-2 interactive/debug cluster: https://docs.hpc.ugent.be/interactive_debug/.

**Review by boegel — CHANGES_REQUESTED** [review]

File: source/gent/tier1_hortense.rst (hunk 1 above)
Comment (stdweird) [reply source/gent/tier1_hortense.rst:56]: ok, but these are indeed VMs

**Comment by stdweird** [comment]
@boegel remarks addressed, thx!

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**